README.md

```diff
@@ -1,2 +1,137 @@
-# apisix-seed
-Do service discovery on the CP side
+# APISIX-Seed for Apache APISIX
+[![Go Report Card](https://goreportcard.com/badge/github.com/api7/apisix-seed)](https://goreportcard.com/report/github.com/api7/apisix-seed)
+[![Build Status](https://github.com/api7/apisix-seed/workflows/unit-test-ci/badge.svg?branch=main)](https://github.com/api7/apisix-seed/actions)
+[![Codecov](https://codecov.io/gh/api7/apisix-seed/branch/main/graph/badge.svg)](https://codecov.io/gh/api7/apisix-seed)
+
+Do service discovery for Apache APISIX on the Control Plane
+
+# What's APISIX-Seed
+[Apache APISIX](https://github.com/apache/apisix) is a dynamic, real-time, high-performance API gateway.
+
+In terms of architecture design, Apache APISIX is divided into two parts: data plane and control plane. The data plane is Apache APISIX itself, which is the component of the traffic proxy and offers many full-featured plugins covering areas such as authentication, security, traffic control, serverless, analytics & monitoring, transformation and logging.
+The control plane is mainly used to manage routing, and implement the configuration center through etcd.
+
+For cloud-native gateways, it is necessary to dynamically obtain the latest service instance information (service discovery) through the service registry. Currently, Apache APISIX already supports [service discovery](https://github.com/apache/apisix/blob/master/docs/en/latest/discovery.md) in the data plane.
+
+This project is a component of Apache APISIX to implement service discovery in the control plane. It supports cluster deployment. At present, we have supported zookeeper and nacos. We will also support more service registries.
+
+The following figure is the topology diagram of APISIX-Seed deployment.
+
+![apisix-seed overview](docs/assets/images/apisix-seed%20overview.png)
```

Comment:
Do you need to use relative paths? The image is not being shown on the preview.

Reply:
I don't quite understand what you mean, this is a relative path. You want me to change to:
```
![apisix-seed overview](./docs/assets/images/apisix-seed%20overview.png)
```
?

Reply:
Yes. This is what I see on the preview now:

![Screenshot 2022-04-27 at 4 16 47 PM](https://user-images.githubusercontent.com/49474499/165501841-c9d29d6a-8c37-49ba-80b2-5b75be759fbf.png)